saas/static/js/djaodjin-saas-angular.js

```diff
@@ -649,3 +674,102 @@ metricControllers.controller('metricCtrl',
             $scope.balances = data;
     });
 }]);
+
+revenueControllers.controller('revenueCtrl',
+    ['$scope', '$http', '$filter', 'urls',
+    function($scope, $http, $filter, urls) {
+
+    $scope.endOfMonth = function(date) {
+        return new Date(
+            date.getFullYear(),
+            date.getMonth() + 1,
+            0
+        )
+    };
+
+    $scope.ends_at = $scope.endOfMonth(new Date());
+
+    $scope.tabs = [
+        {key: 'amount', title: 'Amount', unit: '$'},
+        {key: 'customers', title: 'Customers'},
+    ];
+    $scope.activeTab = 'amount';
```

Comment:
Please see https://github.com/marmida/djaodjin-saas/commit/fad4af51ff298f1145b5897cb513f92962020b35 and the associated private repo patch.